### DISCUSSION TITLE
Emails not being formatted correctly

### DISCUSSION THREAD

**Discussion by Bahpu** [emailjs-com/help#143] [Q&A]
Hi! 

I've been trying to format the email to make sure it displays each detail line by line like this:

```
<p>New request from {{from_name}}:</p>
<div style="padding: 12px; border-left: 4px solid #d0d0d0; font-style: italic; display: flex; justify-content: center; flex-direction: column; row-gap: 15px;">
<div>Reservation Date: {{date}}</div> <br />
<div>Full Name: {{name}}</div> <br />
<div>Email: {{email}}</div> <br />
<div>Phone Number: {{phone_number}}</div> <br />
<div> Reservation Time: {{reservation_time}} </div> <br/>
<div>Request: {{email_body}}</div> <br />
</div>
```

But the email appears like this:

![chrome_7DJ06ABhXp](https://github.com/user-attachments/assets/2ebd18b0-2650-43c1-8e13-feb0413169ef)

I've tried a range of things like using breaks, using without breaks just a flex box etc. I just can't figure out why its still on the same line. I'd appreciate any help! thank you

**Comment by xr0master** [comment]
Unfortunately, I can't tell you the reason. For me, it looks fine.
You can look at the code of your email and try to find the reason there.

I will close it currently, but you can reopen.
